### ISSUE TITLE
Add colors for Neovim `:terminal`

### ISSUE THREAD

**Issue by sbdchd** [jacoborus/tender.vim#14]
There are no colors defined for Neovim's terminal, so it uses the less than ideal defaults.

See [onedark.vim](https://github.com/joshdick/onedark.vim/blob/master/colors/onedark.vim#L439-L456) for an example.

**Comment by jacoborus** [comment]
That's a nice featured I didn't think about. Thank you

**Comment by metalelf0** [comment]
These should be the colors:

```
let g:terminal_color_background =  "#1e1e1e"
let g:terminal_color_foreground =  "#adadad"
let g:terminal_color_0 =  "#1e1e1e"
let g:terminal_color_1 =  "#b60024"
let g:terminal_color_2 =  "#8e9d05"
let g:terminal_color_3 =  "#c8ac74"
let g:terminal_color_4 =  "#63c3f1"
let g:terminal_color_5 =  "#c76fbd"
let g:terminal_color_6 =  "#259286"
let g:terminal_color_7 =  "#eaeaea"
let g:terminal_color_8 =  "#262626"
let g:terminal_color_9 =  "#ee1c42"
let g:terminal_color_10 =  "#bec94a"
let g:terminal_color_11 =  "#fdb63b"
let g:terminal_color_12 =  "#a5d6eb"
let g:terminal_color_13 =  "#e24d8e"
let g:terminal_color_14 =  "#00b39e"
let g:terminal_color_15 =  "#eaeaea"
```

**Comment by jacoborus** [comment]
@metalelf0 thanks, but Tender is generated by [Estilo](https://github.com/jacoborus/estilo) so I need to fix jacoborus/estilo#18 before

**Comment by metalelf0** [comment]
Yes you're right, I converted colors from the iterm scheme someone posted on reddit and thought this can be a useful workaround for someone while you fix it :) Keep up the great work, Estilo is gonna rock! ;)
